src/objective-c/ProtoRPC/ProtoService.m

```diff
@@ -69,9 +69,9 @@ - (ProtoRPC *)RPCToMethod:(NSString *)method
            requestsWriter:(GRXWriter *)requestsWriter
             responseClass:(Class)responseClass
        responsesWriteable:(id<GRXWriteable>)responsesWriteable {
-  ProtoMethod *methodName = [[ProtoMethod alloc] initWithPackage:_packageName
-                                                         service:_serviceName
-                                                          method:method];
+  GRPCProtoMethod *methodName = [[GRPCProtoMethod alloc] initWithPackage:_packageName
+                                                                 service:_serviceName
+                                                                  method:method];
   return [[ProtoRPC alloc] initWithHost:_host
```

Comment:
Shouldn't grpc also be returning instances of GRPCProtoCall? The compiler generates methods that say GRPCProtoCall is returned, but in actuality it is returning a superclass. Caused some strange behavior for us because of some category methods we added.